extensions/npm/package.json

```diff
@@ -237,6 +236,7 @@
           "type": "boolean",
           "default": false,
           "scope": "resource",
+          "deprecationMessage": "The NPM Script Explorer is now available in 'Views' menu in the Explorer in all folders.",
```

Comment:
Wording? The NPM Script Explorer is now available in **the** 'Views' menu in the Explorer in all folders."

Reply:
Fixed, thank you for catching.